app/presenters/candidate_interface/application_form_sections.rb

```diff
@@ -17,6 +29,18 @@ def completed?(section_name)
 
   private
 
+    def primary_course?
+      application_choice.current_course.primary_course?
+    end
+
+    def science_gcse?
+      ->(section) { section.name == :science_gcse }
+    end
+
+    def incomplete_sections?
```

Comment:
Worht having this method? Seems to be only called once.

Reply:
Yes I agree. Deleted